plopfile.js

```diff
@@ -82,4 +65,37 @@ module.exports = function(plop) {
       ];
     },
   });
+
+  plop.setGenerator('tsconfig', {
+    description: 'Generate package level tsconfig',
```

Comment:
Isn't this a root-level ts config. To me, `package level` means for an individual package.

Reply:
Yeah I meant "packages" level since that's the folder it's in, but maybe root makes sense anyway.